includes/data-stores/class-wc-customer-download-data-store.php

```diff
@@ -274,13 +274,17 @@ public function get_downloads( $args = array() ) {
 	/**
 	 * Update download ids if the hash changes.
 	 *
+	 * @deprecated 3.3.0 Download id is now a static UUID and should not be changed based on file hash.
+	 *
 	 * @param  int $product_id
 	 * @param  string $old_id
 	 * @param  string $new_id
 	 */
 	public function update_download_id( $product_id, $old_id, $new_id ) {
 		global $wpdb;
 
+		wc_deprecated_function( 'update_download_id', '3.3' );
```

Comment:
Ah, good point; I'll switch that.  On the replacement, I don't think there's really a replacement in this case since we shouldn't really be changing IDs at all anymore in the new approach.  So instead of replacing calls to this, they simply shouldn't be trying to change IDs.